databricks-sdk-java/src/test/java/com/databricks/sdk/integration/SqlIT.java

```diff
@@ -22,8 +22,10 @@ void listQueryHistoryTimeRange(WorkspaceClient w) {
         new ListQueryHistoryRequest()
             .setFilterBy(new QueryFilter().setQueryStartTimeRange(timeRange));
     Iterable<QueryInfo> queries = w.queryHistory().list(request).getRes();
-    for (QueryInfo query : queries) {
-      System.out.println(query);
+    if (queries != null) {
```

Comment:
Note: We can't use the legacy query API here since `ListQueriesLegacyRequest` doesn't support `setFilterBy`